src/StatsBase.jl

```diff
@@ -9,7 +9,7 @@ module StatsBase
 
     import SpecialFunctions: erfcinv
 
-    using Compat
+    using Compat, SortingAlgorithms
```

Comment:
SortingAlgorithms will need to be added to REQUIRE.

Reply:
Hehe, I checked too, and it's already there...

Reply:
Oh, weird.

Reply:
Ping! I have made all the changes. Please review.

Reply:
SortingAlgorithms is not in REQUIRE... Causing package to fail to load right now.

Reply:
Woops. But how is it possible that tests passed, given that they exercize the new code which relies on it?

Reply:
DataFrames which is a test dependency pulls SortingAlgorithms

Reply:
Indeed.

@xiaodaigh Would you make a pull request to add SortingAlgorithms to REQUIRE?

Reply:
Shashi already did so